HOTT-990: Removed the alerting for missing flags

jebw opened the pull request:
### Jira link

[HOTT-990](https://transformuk.atlassian.net/browse/HOTT-990)

### What?

I have added/removed/altered:

- [x] Removed the alerting for missing flag images

### Why?

I am doing this because:

- GeographicalArea's like XI don't actually have flags so this will just alert indefinitely without being fixable.

alexdesi commented on app/helpers/country_flag_helper.rb:
```diff
@@ -3,9 +3,6 @@ def country_flag_tag(two_letter_country_code, **kwargs)
   rescue Webpacker::Manifest::MissingEntryError
-    Raven.capture_message \
-      "Missing flag image file for #{two_letter_country_code.downcase}"
-
```
Not sure how often this happens, maybe should we just log it, as a info?

jebw committed:
HOTT-990: Skip alerting for countries without flag

Some countries do not have a flag so we should
not alert Sentry for those countries

alexdesi commented on spec/helpers/country_flag_helper_spec.rb:
```diff
@@ -35,5 +35,17 @@
       end
     end
+
+    context "with country which we don't have a flag which is in ignore list" do
```
this should be a bit more readable:
`context 'when country do not have a flag and is in the ignore list'`

alexdesi reviewed (approved):
👍
